README.md

```diff
@@ -1,3 +1,17 @@
+## Quick reference
+
+Maintained by: [the ROOT team](https://root.cern/) and collaborators
+
+Where to get help: [the ROOT forum](https://root-forum.cern.ch/)
+
+## Supported tags and respective Dockerfile links
+
+* [`latest`](https://github.com/root-project/root-docker/blob/6.20.06-fedora32/fedora/Dockerfile), [`6.20.06-fedora32`](https://github.com/root-project/root-docker/blob/6.20.06-fedora32/fedora/Dockerfile)
+* [`6.22.00-conda`](https://github.com/root-project/root-docker/blob/6.22.00-conda/conda/Dockerfile)
+* [`6.20.06-arch`](https://github.com/root-project/root-docker/blob/6.20.06-arch/conda/Dockerfile)
+* [`6.20.06-centos7`](https://github.com/root-project/root-docker/blob/6.20.06-centos7/cc7/Dockerfile)
+* [`6.22.00-ubuntu20.04`](https://github.com/root-project/root-docker/blob/6.22.00-ubuntu20.04/ubuntu/Dockerfile)
```

Comment:
See also https://hub.docker.com/r/rootproject/root/dockerfile. I don't know if it's possible, but maybe we can make this point to the right file for each tag and link to that instead?

Reply:
as far as I can tell that page is per image, not per tag?